.github/workflows/develop.yml

```diff
@@ -47,26 +39,13 @@ jobs:
             with:
                 python-version: ${{ matrix.python-version }}
 
-        -   name: Install system dependencies
+        -   name: Install Python dependencies
             run: |
-                sudo apt update
-                sudo apt install postgresql-12
-
-        -   name: Install python dependencies
-            continue-on-error: true
-            id: install
-            run: |
-                pip install --upgrade setuptools
+                pip install --upgrade pip setuptools wheel
                 pip install git+https://github.com/aiidateam/aiida-core@develop#egg=aiida-core[atomic_tools]
                 pip install -e .[tests]
                 reentry scan
```

Comment:
Do we still have to run `reentry scan` for the `develop` branch of `aiida-core`?

Reply:
Absolutely, removed it. This reminds me though that merging this now might not be a good idea, as it won't work as anticipated. The install from source of `aiida-core` will work but `pip install -e .[tests]` will then install with requirements of our own develop branch and that still specified something like `aiida-core~=1.6` and so it will uninstall the `develop` branch of `aiida-core`. So we need a v2.0 release of `aiida-core` and then up our own requirements.